client/viame-web-common/components/RunPipelineMenu.vue

```diff
@@ -14,53 +16,70 @@ export default {
     },
   },
 
-  computed: {
-    ...mapState('Pipelines', ['pipelines']),
-    selectedEligibleClips() {
-      return this.selected.filter(
-        ({ _modelType, meta }) => _modelType === 'folder' && meta && meta.annotate,
-      );
-    },
-    pipelinesNotRunnable() {
-      return this.selectedEligibleClips.length < 1 || this.pipelines === null;
-    },
-  },
+  setup(props, { root }) {
+    const { runPipeline, getPipelineList } = useApi();
+    const unsortedPipelines = ref({} as Pipelines);
+    onBeforeMount(async () => {
+      unsortedPipelines.value = await getPipelineList();
+    });
 
-  created() {
-    this.fetchPipelines();
-  },
+    const pipelines = computed(() => {
+      const sortedPipelines = {} as Pipelines;
+      Object.entries(unsortedPipelines.value).forEach(([name, category]) => {
+        category.pipes.sort((a, b) => {
+          const aName = a.name.toLowerCase();
+          const bName = b.name.toLowerCase();
+          if (aName > bName) {
+            return 1;
+          }
+          if (aName < bName) {
+            return -1;
+          }
+          return 0;
+        });
+        sortedPipelines[name] = category;
+      });
+      return sortedPipelines;
+    });
 
-  methods: {
-    ...mapActions('Pipelines', ['fetchPipelines']),
+    const pipelinesNotRunnable = computed(() => (
+      props.selectedDatasetIds.length > 1 || pipelines.value === null
+    ));
 
-    async runPipelineOnSelectedItem(pipeline) {
-      const clips = this.selectedEligibleClips;
+    async function runPipelineOnSelectedItem(pipename: string) {
       await Promise.all(
-        this.selectedEligibleClips.map((item) => runPipeline(item._id, pipeline)),
+        props.selectedDatasetIds.map((id) => runPipeline(id, pipename)),
```

Comment:
This probably is related to other issues.  You expect Id's but you're passing the entire object for each folder not just the ids.  So either restructure the data in `Home.vue` and `ViewerLoader.vue` to handle ids or revert this back to extracting the `_id` from the object.  In Home.vuew you pass in the selectedViameFolders, so you may want a computed property to gather the selected folder ids or could accept doing the extraction in here.